packages/webdriverio/src/utils/interception/devtools.ts

```diff
@@ -262,6 +284,12 @@ export default class DevtoolsInterception extends Interception {
     abortOnce (errorReason: Protocol.Network.ErrorReason) {
         this.abort(errorReason, false)
     }
+
+    private ensureNotRestored() {
+        if (this.restored) {
+            throw new Error('This can\'t be done on restored mock')
+        }
+    }
```

Comment:
If somebody has tests, where `abort` / `respond` is called after `restore`, currently it would work just like creating the new one with mocked response.

After this commit, `abort` / `respond` after `restore` just won't do anything, so i added this check to let people know, that you can't do that now.

Also, should i include `BREAKING CHANGE` in commit message footer? That could be considered as breaking change.